src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    protected String taskName;
+    protected boolean isDone;
+
+    public Task(){
+        this.taskName = "";
+        this.isDone = false;
+    }
+
+    public Task(String task){
+        this.taskName = task;
+        this.isDone = false;
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public boolean getDone(){
```

Comment:
checkIsDone may be a better name to reflect the boolean nature of the variable to get